tcf_website/static/course/javascript/course_data.js

```diff
@@ -96,7 +88,11 @@ const createChart = gradesData => {
                     label: function(tooltipItem, data) {
                         var dataset = data.datasets[0];
                         var percent = Math.round((dataset.data[tooltipItem.index] / dataset._meta[0].total) * 100);
-                        return data.labels[tooltipItem.index] + ": " + percent + "%";
+                        var label = data.labels[tooltipItem.index];
+                        if (tooltipItem.index === 9) {
+                            label = "No Credit";
```

Comment:
Will `No Credit` be enough for users? I agree it's the most accurate term, but I'm afraid it might not be the most intuitive one.

Reply:
I thought the same, but I'm not sure where to add an elaboration on what it means because you'd have to have like a tooltip on a tooltip or have it somewhere not on the chart. Any ideas?

Reply:
I thought it would be okay to add more in the parentheses following that, but we should probably ask the team.